src/components/common/SearchBar.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { term: '' };
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onFormSubmit = this.onFormSubmit.bind(this);
+  }
+
+  onInputChange(event) {
+    this.setState({ term: event.target.value });
+  }
+  onFormSubmit(event) {
+    event.preventDefault();
+    //Need to call an action here to update application state 'searchQuery'
+    console.log('You typed ', this.state.term);
```

Comment:
Console logs only allowed for personal development, should never be part of a PR